label_studio/tasks/models.py

```diff
@@ -66,7 +66,12 @@ class Task(TaskMixin, models.Model):
         help_text='Uploaded file used as data source for this task'
     )
     updates = ['is_labeled']
-
+    total_annotations = models.IntegerField(_('total_annotations'), default=0, db_index=True,
+                                  help_text='Number of total annotations for the current task')
```

Comment:
'Number of total annotations for the current task except cancelled annotations'

Reply:
Fixed